tests/v2/validation/rbac/rbac_test.go

```diff
@@ -0,0 +1,239 @@
+	require.NoError(rb.T(), err)
+	fmt.Printf("Created project as a %s: %s. \n",role,createProjectAsClusterMembers.Name)
+	require.NoError(rb.T(), err)
+	actualStatus := fmt.Sprintf("%v", createProjectAsAdmin.State)
+	assert.Equal(rb.T(),"active",actualStatus)
+	fmt.Println("Validation successful!")
+
+	//Testcase4 Validate if cluster members can create namespaces in project they are not owner of
+	fmt.Printf("Testcase4 - Validating if %s can create namespace in a project they are not owner of. \n",role)
+	namespaceName := AppendRandomString("testns-")	
+	createdNamespace, err := namespaces.CreateNamespace(rb.standardUserClient, namespaceName, "{}", map[string]string{}, map[string]string{}, createProjectAsAdmin)
+	if role == "cluster-owner"{
+		require.NoError(rb.T(), err)
+		fmt.Println("Created a namespace as cluster Owner: ",createdNamespace.Name)
+		assert.Equal(rb.T(), namespaceName, createdNamespace.Name)
+		actualStatus = fmt.Sprintf("%v", createdNamespace.Status.Phase)
+		assert.Equal(rb.T(), "Active", actualStatus)
+		fmt.Println("Validation successful!")
+	}else if role == "cluster-member"{
+		require.Error(rb.T(), err)
+		//assert cluster member gets an error when creating a namespace in a project they are not owner of
+		errMessage := strings.Split(err.Error(), ":")[0]
+		assert.Equal(rb.T(),"namespaces is forbidden",errMessage)
+		fmt.Println("Validation successful!")	
+	}
+
+	//Testcase5 Validate if cluster members are able to list all the namespaces in a cluster
+	fmt.Printf("Testcase5 - Validating if %s can lists all namespaces in a cluster. \n",role)
+	//Get the list of namespaces as and admin client
+	namespaceListAdmin, err := GetListNamespaces(rb.client, rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	//Get the list of namespaces as an admin client
+	namespaceListClusterMembers, err := GetListNamespaces(rb.standardUserClient, rb.cluster.ID)
+	if role == "cluster-owner"{
+		require.NoError(rb.T(), err)
+		//Length of namespace list for admin and cluster owner should match
+		assert.Equal(rb.T(), len(namespaceListAdmin), len(namespaceListClusterMembers))
+		//Namespaces obtained as admin and cluster owner should be same
+		assert.Equal(rb.T(), namespaceListAdmin, namespaceListClusterMembers)
+		fmt.Println("Validation successful!")
+	}else if role == "cluster-member"{
+		require.Error(rb.T(), err)
+		errMessage := strings.Split(err.Error(), ":")[0]
+		//assert namespaces list obtained as a cluster member is empty
+		assert.Equal(rb.T(), len(namespaceListClusterMembers), 0)
+		//assert cluster member is not able to list namespaces
+		assert.Equal(rb.T(), "namespaces is forbidden",errMessage)
+		fmt.Println("Validation successful!")
+	}
+
+	//Testcase6 Validate if cluster members are able to delete the project they are not owner of
+	fmt.Printf("Testcase6 - Validating if %s can delete a project they are not owner of. \n",role)
+	err = projects.DeleteProject(rb.standardUserClient, createProjectAsAdmin)
+	if role == "cluster-owner"{
```

Comment:
Fix the formatting on the if/else if statements. The blocks aren't indented and there is no space between the statement and the `{` it's hard to read